Cython/Compiler/ModuleNode.py

```diff
@@ -3109,7 +3109,7 @@ def generate_module_init_func(self, imported_modules, env, code):
         # user code in atexit or other global registries.
         ##code.put_decref_clear(env.module_dict_cname, py_object_type, nanny=False)
         code.putln('}')
-        code.putln("#if !CYTHON_COMPILING_IN_LIMITED_API")
+        code.putln("#if !CYTHON_USE_MODULE_STATE")
```

Comment:
I also changed this (around the cleanup of `__pyx_m`) since it seemed more tied to `CYTHON_USE_MODULE_STATE` than to limited API specifically.